ios/FluentUI/Command Bar/CommandBar.swift

```diff
@@ -28,9 +28,14 @@ open class CommandBar: UIView {
 
     @objc public init(itemGroups: [CommandBarItemGroup], leadingItem: CommandBarItem? = nil, trailingItem: CommandBarItem? = nil) {
         self.itemGroups = itemGroups
+        self.leadingItem = leadingItem
+        self.trailingItem = trailingItem
 
         super.init(frame: .zero)
+        updateView()
+    }
 
+    private func updateView() {
```

Comment:
nit: order of members in the file (private should be down below)

https://github.com/microsoft/swift-guide/blob/gh-pages/FileOrganization.md